packages/slice-machine/src/features/builder/StaticFieldsInfoDialog.tsx

```diff
@@ -0,0 +1,78 @@
+import {
+  Box,
+  Dialog,
+  DialogActionButton,
+  DialogActions,
+  DialogContent,
+  DialogHeader,
+  ScrollArea,
+  Text,
+  Video,
+} from "@prismicio/editor-ui";
+import { ReactNode, useState } from "react";
+
+interface StaticFieldsInfoDialogProps {
+  onClose: () => void;
+  trigger?: ReactNode;
+}
+
+export function StaticFieldsInfoDialog(props: StaticFieldsInfoDialogProps) {
+  const { onClose, trigger } = props;
+  const [isOpen, setIsOpen] = useState(false);
+
+  const handleOpenChange = (open: boolean) => {
+    if (!open) {
+      dismissDialog();
+    } else {
+      setIsOpen(open);
+    }
+  };
+
+  function dismissDialog() {
+    setIsOpen(false);
+    onClose();
+  }
+
+  return (
+    <Dialog
+      open={isOpen}
+      onOpenChange={handleOpenChange}
+      trigger={trigger}
+      size="small"
+    >
+      <DialogHeader title="Static zone" />
+      <DialogContent>
+        <ScrollArea>
+          <Box as="article" flexDirection="column" padding={16} gap={4}>
+            <Text sx={{ marginBottom: 4 }} color="purple9" variant="smallBold">
+              About
+            </Text>
+            <Text variant="h3">What is the Static zone</Text>
+            <Text color="grey11">
+              The static zone in Prismic page types contain fields that are
+              always present on the page. Use it for essential information like
+              a page's title, page design settings or metadata. Unlike slices,
+              which can be added or removed, static zone fields remain fixed.
+            </Text>
+            <Video
+              src="https://res.cloudinary.com/dmtf1daqp/video/upload/v1723540305/DEV_TOOLS/STATES/Explication_vide%CC%81o_zone_statique_phibeq.mp4"
+              sizing="contain"
+              autoPlay
+              loop
+            />
+          </Box>
+        </ScrollArea>
+      </DialogContent>
+      <DialogActions>
+        <DialogActionButton
+          size="medium"
+          sx={{ flexGrow: 1 }}
+          onClick={dismissDialog}
+          data-testid="static-fields-info-dialog-cta"
+        >
+          Got it
```

Comment:
❓#question

Even if you never click on "Got it" you will not see it again after, is that wanted?

Reply:
Not sure I understood, if you dismiss the modal it won't be opened again on add static field.

Reply:
What I mean is when you don't click on "Got it" but close the modal with another way like the close icon, it will still be taken as "Got it". Just wanted to be sure it's ok for product.

Reply:
Ah, will check that

Reply:
Checked with Benj & Côme and now closing the dialog doesn't prevent it from showing up the next time